ci/docker/nuxeo/templates/nuxeo-js-client-test/nxserver/config/nuxeo-js-opensearch-config.xml.nxftl

```diff
@@ -0,0 +1,39 @@
+<#escape x as x?xml>
+<?xml version="1.0"?>
+<component name="org.nuxeo.test.opensearch">
+
+<#if (nuxeo.search.client.default.name)?has_content>
+  <extension target="org.nuxeo.runtime.opensearch1.OpenSearchComponent" point="index">
+    <index name="other">
+      <client id="search/default" />
+      <mapping file="opensearch1-doc-mapping.json" />
+      <settings file="opensearch1-doc-settings.json" />
+    </index>
+  </extension>
+
+  <extension target="org.nuxeo.ecm.core.search.client.opensearch1" point="searchClient">
+    <searchClient name="opensearch">
+      <searchIndex name="enhanced-other" technicalName="other" />
+    </searchClient>
+  </extension>
+
+  <extension target="org.nuxeo.ecm.core.search" point="searchIndex">
+    <searchIndex name="enhanced-other" searchClient="opensearch" repository="other" default="true" />
+  </extension>
```

Comment:
Not sure it is needed, I didn't find any test using the search on the other repository with OpenSearch.

Reply:
Yes same.